docs/src/main/asciidoc/validation.adoc

```diff
@@ -474,6 +474,19 @@ In this case, it means `Book.id` must be `null` and `Book.title` must not be bla
 will be validated for the `book` parameter of the `put` method.
 In this case, it means `Book.id` must not be `null` and `Book.title` must not be blank.
 
+== Limitations
+
+=== META-INF/validation.xml
+
+Configuring the `ValidatorFactory` using a `META-INF/validation.xml` file is *NOT* supported in Quarkus.
```

Comment:
So it will just be ignored silently?

Do you think it would be a good idea to fail on startup if that file exists, and provide a configuration option `quarkus.hibernate-validator.validation-xml.ignore`, like we did for `persistence.xml`?

Reply:
Dunno. Might be a good idea but it might also break some expectations. I'll merge that one and we can iterate later if we think it's the way to go.